### PR TITLE
Endpoint for removing a registry

### LINKED ISSUE #85: Endpoint to remove a registry and reset the list of registries

### PR DESCRIPTION
**Pull request by nsosio**
Fixes #85 and #112

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -74,6 +74,28 @@ async def add_registry(body: schemas.RegistryInput):
         ) from error
 
 
+@router.post(
```
Comment (filopedraz) [inline app/routes.py:77]: Instead of a POST, I would have used a DELETE passing the `registry_url` as URL parameter.

**Review by filopedraz — CHANGES_REQUESTED** [review]

File: app/routes.py (hunk 1 above)
Comment (nsosio) [reply app/routes.py:77]: @filopedraz I'll update it

Comment (filopedraz) [reply app/routes.py]: Resolve the conversation when handled :)

**Comment by filopedraz** [comment]
@nsosio some conflicts, you need to rebase. Good job

**Comment by nsosio** [comment]
@filopedraz fixed :)

**Pull request merged**